addons/delete-others/addon.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Delete other assets",
+  "description": "Adds an option to the right click menu to easily delete all other costumes, backdrops, or sounds.",
+  "tags": ["editor", "costumeEditor", "recommended"],
```

Comment:
Recommended? @mxmou @Joeclinton1

Reply:
I think we forgot to look at the tags

Reply:
It should probably be featured but not recommended, like `move-to-top-bottom`.

Reply:
Ok, I opened PR #7733